src/SFA.DAS.AssessorService.Api.Types/Models/Certificates/CertificateSummaryResponse.cs

```diff
@@ -51,4 +50,4 @@ public class CertificateStatusList
         public DateTime CertificateDate { get; set; }
```

Comment:
The CertificateStatusList remains here - is it required? this seems to have been removed almost completely, I think it was never in the AC's has it just been dropped as a mistake?

Reply:
removed